docs/decisions/.markdownlint.yml

```diff
@@ -0,0 +1,4 @@
+default: true
+MD013: false
+MD024: false
+MD040: false
```

Comment:
Can you please specify in a comment why these rules has been disabled?

Reply:
It can be difficult to make ADR format consistent with our linting rules.
Anyway, I will try to investigate a bit more to understand what rules we have to necessarily relax here.